public_html/index.php

```diff
@@ -177,9 +177,9 @@
 
 <span class="text-white-50"># Launch the RNAseq pipeline</span>
 nextflow run nf-core/rnaseq \
-    -profile docker \
+    --input samplesheet.csv \
```

Comment:
Is that change already released in the rnaseq pipeline?

Reply:
Good point 👍 Not yet but imminent (he says). Can leave this open if you like?